db/migrate/20150606065713_add_sti_to_lookup_keys.rb

```diff
@@ -0,0 +1,27 @@
+class AddStiToLookupKeys < ActiveRecord::Migration
+  def up
+    add_column :lookup_keys, :type, :string
+    LookupKey.scoped.each do |key|
+      if key.is_param
+        key.type = 'PuppetclassLookupKey'
```

Comment:
I don't remember why it didn't work, but it didn't work, that was my first iteration on this migration, as it's obviously what should be done (and is done in the down...)